server/tm_cmds.go

```diff
@@ -16,26 +15,6 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// ShowNodeIDCmd - ported from Tendermint, dump node ID to stdout
-func ShowNodeIDCmd() *cobra.Command {
-	return &cobra.Command{
-		Use:   "show-node-id",
-		Short: "Show this node's ID",
-		RunE: func(cmd *cobra.Command, args []string) error {
-			serverCtx := GetServerContextFromCmd(cmd)
```

Comment:
> utACK. I hope these work. @marbar3778 did you test them? I know Tendermint has some wonky usage of globals in their commands, specifically with how the config file is fetched and read. I recall having to open a PR to fix one of their commands.

Agree with @alexanderbez, I also saw some weird stuff while using TM cmds directly. I think it's because in the SDK ports, we have this `GetServerContextFromCmd` which fixes some flags parsing for the SDK.

If it's tested manually, then OK to merge.

Reply:
did some simple testing but need to test different home directories

Reply:
https://github.com/tendermint/tendermint/blob/v0.34.21/cmd/tendermint/commands/reindex_event.go#L54

You can see here it relies on a global `config` :(

Reply:
gonna wait on tendermint to fix, give it a few days if nothing ill wrap them

Reply:
Sounds good 👍